package.json

```diff
@@ -28,6 +31,16 @@
     "inquirer": "^9.1.5"
   },
   "devDependencies": {
-    "prettier": "^2.8.7"
```

Comment:
We should keep _prettier_. 

ESLint will be used to check syntax and find problems, while Prettier will be used to enforce code style

Reply:
Done adding it